.github/workflows/tests.yaml

```diff
@@ -16,13 +14,15 @@ jobs:
         - linux-amd64-integration-4-cpu
         - linux-amd64-functional
         - linux-amd64-unit-4-cpu-race
+        - linux-amd64-e2e
+        - linux-386-e2e
         - all-build
         - linux-386-unit-1-cpu
     steps:
     - uses: actions/checkout@v2
     - uses: actions/setup-go@v2
       with:
-        go-version: ${{ matrix.go }}
+        go-version: 1.16.3
```

Comment:
Should we `1.16.4`?

Reply:
Let's move update of go version to a separate PR, as this applies to a multiple files: 

```
.github/workflows/tests.yaml:        go: [1.16.3]
.travis.yml:  - 1.16.3
.travis.yml:    - go: 1.16.3
.travis.yml:    - go: 1.16.3
Makefile:GO_VERSION ?= 1.16.3
```